segmentation/include/pcl/segmentation/extract_clusters.h

```diff
@@ -434,6 +434,4 @@ namespace pcl
   }
 }
 
-#ifdef PCL_NO_PRECOMPILE
```

Comment:
I don't really understand why you removed this (and in extract_labeled_clusters.h)?

Reply:
The hpp file needs to be present so that the cpp can include just the main header. This flag makes the prev proposition conditional

Reply:
Both hpp files include the corresponding h files, so if the cpp includes the hpp files, everything should be there, no? Most if not all other classes follow that pattern.

Reply:
The public API is supposed to be the `.h` file, not the `.hpp` file

Reply:
I undid those changes